### PR TITLE
[feature #156577841] Create dashboard for failed checks

### PR DESCRIPTION
**Pull request by hariclerry**
#### What does this PR do?
Have dashboard for viewing failed checks working.

#### Description of Tasks to be complete
 The system should enable user to have a separate dashboard to view failed jobs.

#### How should this be manually tested?
1: Install the project and install the requirements.txt
2.Set up a super user account by running this command; ./manage.py createsuperuser
3: Start the server by running; python manage.py runserver, open localhost  http://localhost:8080 in the browser
4: Login using your email and password you set up in number 2 above
5: Create a job/check
6: Run the job by pinging the check URL and wait for it to fail
7: Click on the failed checks link
8: You should be able to see the failed checks appearing on the page.
#### What are the relevant pivotal tracker stories?
#156577843
#### Checklist: I completed these to help reviewers
- [x] My IDE is configured to follow the PEP8 Conventions

No? Unsure? -> configure your IDE, format the code and add the changes with git add . && git commit --amend
- [x] I have added tests to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

No? -> write tests and add them to this commit git add . && git commit --amend

- [x] All new and existing tests passed.

No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is based on the latest changes of the develop branch.

No? Unsure? -> execute command git pull --rebase origin develop

### CODE REVIEW

**Comment by hariclerry** [comment]
Hello Team, please review my code and let me know what you guys think.

File: hc/settings.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
 HOST = "localhost"
-SECRET_KEY = os.environ.get("HC_SECRET_KEY")
+SECRET_KEY = '---'
```
Comment (aenkya) [inline hc/settings.py]: why are you putting a hard coded secret key

**Review by aenkya — CHANGES_REQUESTED** [review]

File: hc/settings.py (hunk 1 above)
Comment (aenkya) [reply hc/settings.py]: maintain the PR title convention.
Also add the checklist of things to do for the PR to the PR description

Comment (hariclerry) [reply hc/settings.py]: Oh, forgot to change it. I will fix it.

Comment (hariclerry) [reply hc/settings.py]: I have fixed it.

**Pull request merged**